File: ksRoundA.cpp
#include <iostream>
using namespace std;

int main(){
	int testCase, n, b;
	int sortArr(int, int);
	int maxB = (10^5);
	cin>>testCase;
	if (testCase >= 1 && testCase <= 100){
		cin>>n>>b;
		int costs[maxB];
		for (int i = 1; i <= n; i ++){
			cin>>costs[i];
			sortArr(costs, n);
		}
	}
	return 0;
}

int sortArr(int arr[], int n){
	int temp, i, j;
	for(i = 0; i < n; i ++)
	{
		for(j = i + 1; j < n; j ++)
		{
			if(arr[i] > arr[j])
			{
				temp = arr[i];
				arr[i] = arr[j];
				arr[j] = temp;
			}
		}
	}
	return arr[];
}

File: ksRoundA.py
testCase = int(input())
for j in range(testCase):
    if (testCase >= 1 and testCase <= 100):
        # n = int(input())
        # budget = int(input())
        n, budget = [int(x) for x in input().split()[:2]]
        if (n >= 1 and n <= (10 ** 5)):
            costs = [int(x) for x in input().split()[:n]]
            # for i in range(0, n):
            #     element = int(input())
            #     costs.append(element)
            costs.sort()
            temp = []
            for i in range(0, n):
                if (costs[i] <= budget):
                    if (sum(temp) <= budget):
                        if (sum(temp) + costs[i] <= budget):
                            temp.append(costs[i])
            print("Case #{}: {}".format(j+1, len(temp)))
